t/058-tcp-socket.t

```diff
@@ -4367,3 +4367,64 @@ connect failed: missing the port number
 finish
 --- no_error_log
 [error]
+
+
+
+=== TEST 73: run in init_worker_by_lua
```

Comment:
What makes more sense is to test cases for the `init_by_lua*` and `exit_worker_by_lua*` contexts.